### PR TITLE
Version script v3

### PR DESCRIPTION
**Pull request by cabelitos**
Changes since v2:
* Added commit  92cda4f  4ad6aa1 and  1e4de6a
* Sorting symbols alphabetically in libsoletta.sym

### CODE REVIEW

File: src/shared/include/sol-file-reader.h [hunk 1]
```diff
@@ -96,3 +100,7 @@ struct sol_blob *sol_file_reader_to_blob(struct sol_file_reader *fr);
 /**
  * @}
  */
+
+#ifdef __cplusplus
+}
+#endif
```
Comment (bdilly) [inline src/shared/include/sol-file-reader.h]: +1

File: src/bin/sol-fbp-runner/main.c [hunk 2]
```diff
@@ -316,9 +309,6 @@ shutdown(void)
     if (the_runner)
         runner_del(the_runner);
 
-    if (str_arena)
-        sol_arena_del(str_arena);
-
```
Comment (bdilly) [inline src/bin/sol-fbp-runner/main.c]: what?

+1

File: src/test/test-lwm2m.c [hunk 3]
```diff
@@ -635,9 +635,9 @@ main(int argc, char *argv[])
     client = sol_lwm2m_client_new(CLIENT_NAME, OBJ_PATH, SMS_NUMBER, objects, NULL);
     ASSERT(client != NULL);
 
-    r = sol_lwm2m_add_object_instance(client, &server_object, NULL);
+    r = sol_lwm2m_client_add_object_instance(client, &server_object, NULL);
     ASSERT_INT_EQ(r, 0);
-    r = sol_lwm2m_add_object_instance(client, &security_object, NULL);
+    r = sol_lwm2m_client_add_object_instance(client, &security_object, NULL);
```
Comment (bdilly) [inline src/test/test-lwm2m.c]: ack

File: src/lib/common/include/sol-platform-linux.h [hunk 4]
```diff
@@ -169,7 +192,7 @@ struct sol_uevent {
 int sol_platform_linux_uevent_subscribe(const char *action, const char *subsystem, void (*cb)(void *data, struct sol_uevent *uevent), const void *data);
 
 /**
- * Unsubscribe @c uevent_cb() for @c action and @c subsystem events monitoring
+ * @brief Unsubscribe @c uevent_cb() for @c action and @c subsystem events monitoring
  *
  * @param action The action we're monitoring and want to release the callback
  * @param subsystem The subsystem we're monitoring and want to release the callback
```
Comment (bdilly) [inline src/lib/common/include/sol-platform-linux.h]: seems ok

File: tools/build/Makefile.vars [hunk 5]
```diff
@@ -314,7 +315,7 @@ COMMON_CFLAGS += -std=gnu99
 LIB_SOLETTA_SHARED_LD_FLAGS := -L$(build_stagedir)shared/ -lsolettashared
 
 EXEC_LDFLAGS := $(COMMON_LDFLAGS)
-LIB_LDFLAGS := $(COMMON_LDFLAGS) $(LIB_SOLETTA_SHARED_LD_FLAGS)
+LIB_LDFLAGS += $(COMMON_LDFLAGS) $(LIB_SOLETTA_SHARED_LD_FLAGS)
```
Comment (bdilly) [inline tools/build/Makefile.vars]: ok

**Comment by bdilly** [comment]
it seems good to be merged

**Comment by cabelitos** [comment]
merged

**Pull request closed without merging**